src/abstract-classes/browser-plugin.js

```diff
@@ -179,6 +192,19 @@ class BrowserPlugin {
 
         return false;
     }
+
+    /**
+     *
+     * @param {string} dir - absolute path to the directory.
+     * @returns {Promise<void>}
+     */
+    async _ensureDir(dir) {
+        try {
+            await fs.ensureDir(dir);
+        } catch (error) {
+            log.debug(`Could not ensure dir: ${dir}`, { error });
+        }
+    }
```

Comment:
Should we really catch this? I mean, if the dir is not ensured, then Firefox launch will fail with a cryptic error. Better to fail fast here.

Reply:
good point.